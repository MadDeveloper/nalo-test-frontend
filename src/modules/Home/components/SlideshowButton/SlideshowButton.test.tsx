import renderer from "react-test-renderer"
import { SlideshowButton } from "."

const onClick = jest.fn()

describe("Home module > SlideshowButton", () => {
  it("should match snapshot", () => {
    const tree = renderer
      .create(
        <div>
          <SlideshowButton onClick={onClick} />
          <SlideshowButton previous onClick={onClick} />
        </div>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
