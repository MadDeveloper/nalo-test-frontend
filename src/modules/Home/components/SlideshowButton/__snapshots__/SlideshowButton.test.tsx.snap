// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home module > SlideshowButton should match snapshot 1`] = `
<div>
  <button
    className="root"
    onClick={[MockFunction]}
  >
    <svg
      aria-hidden="true"
      className="root icon"
      fill="currentColor"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M16.28 11.47a.75.75 0 010 1.06l-7.5 7.5a.75.75 0 01-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 011.06-1.06l7.5 7.5z"
        fillRule="evenodd"
      />
    </svg>
  </button>
  <button
    className="root"
    onClick={[MockFunction]}
  >
    <svg
      aria-hidden="true"
      className="root icon"
      fill="currentColor"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M7.72 12.53a.75.75 0 010-1.06l7.5-7.5a.75.75 0 111.06 1.06L9.31 12l6.97 6.97a.75.75 0 11-1.06 1.06l-7.5-7.5z"
        fillRule="evenodd"
      />
    </svg>
  </button>
</div>
`;
